pkg/k8s/resource/resource.go

```diff
@@ -46,19 +45,18 @@ type Resource[T k8sRuntime.Object] interface {
 	Store(context.Context) (Store[T], error)
 }
 
-// NewResourceConstructor provides a constructor for Resource when given a function
-// that maps from a Clientset into a ListerWatcher:
+// New creates a new Resource[T]. Use with hive.Provide:
 //
-//	var exampleCell = cell.Module(
+//	var exampleCell = hive.Module(
```

Comment:
`Module` and `Provide` are still located in the cell package.

Reply:
Ah right, fixed!